flash/vision/detection/model.py

```diff
@@ -57,25 +62,44 @@ class ObjectDetector(Task):
     def __init__(
         self,
         num_classes: int,
-        model: Union[str, nn.Module] = "fasterrcnn_resnet50_fpn",
+        backbone: Optional[str] = None,
```

Comment:
shall we rather move these at the end, like this breaks previous usage

Reply:
@Borda Actually, ukw, I have decided to add `RetinaNet` in this PR as well. The model could switch from `fasterrcnn` and `retinanet`. Making it Draft.

Reply:
so at least what I see is rename argument `model` >> `backbone`

Reply:
Updated!